### PR TITLE
Recipe: Sending Emails

### PR DESCRIPTION
**Pull request by Tobbe**
I wrote a cookbook recipe on building a newsletter signup form as a way to show how to send emails.

Do we want this as a recipe? Or should I just put in on my own blog and link it in the forums instead? I noticed there was an issue for creating an email recipe, but this isn't *exactly* what's described in that issue. https://github.com/redwoodjs/redwoodjs.com/issues/94

So let me know how you want me to proceed with this

### CODE REVIEW

File: cookbook/Sending_Emails.md [hunk 1]
```diff
@@ -0,0 +1,461 @@
+# Newsletter
+
+A common task many web apps need to do is to send emails. To demonstrate how this can be done with Redwood we're going to build a form that lets a user sign up for a newsletter. When doing so we'll send them a "welcome" email. The emails will be sent using the npm package nodemailer together with [SendInBlue](https://sendinblue.com).
```
Comment (cannikin) [inline cookbook/Sending_Emails.md:3]: Maybe link `nodemailer` to the npm page?

File: cookbook/Sending_Emails.md [hunk 2]
```diff
@@ -0,0 +1,461 @@
...
+
+## Wrap up
+
+This should be it for the limited scope of this project. We started by creating a fresh Redwood project. We decided what our database model should look like. Then we designed a simple form on the front-end. We generated and adapted the code needed to use GraphQL to communicate with our backend api. After creating an account with SendInBlue and getting our SMTP key we built out our backend. The two main things we did was create an email library function and updated the newsletterSubscriber service to use it.
+
+### Further improvements
+
+To be a good Netizen we should add an "Unsubscribe" link to all outgoing newsletter emails, even the initial confirmation email. The initial email should also be a proper confirmation email where the user needs to click a link/button to confirm their email and that they actually do want to subscribe to the newsletter. Failing to do these two things will make Google and other email providers upset, and so they might eventually send your emails to the Spam folder, or even block you.
+
+Obviously could also spend some more time on the design, especially the confirmation part in the gui when clicking on the Subscribe button.
```
Comment (cannikin) [inline cookbook/Sending_Emails.md:455]:
```suggestion
Obviously we could also spend some more time on the design, especially the confirmation part in the gui when clicking on the Subscribe button.
```

File: cookbook/Sending_Emails.md [hunk 3]
```diff
@@ -0,0 +1,461 @@
...
+
+Like all demo apps we're also missing error handling. What happens for example if you try to subscribe again with an already subscribed email address?
+
+### Thanks for reading
+
+Thanks for reading this! If you liked it, or have any questions, don't hesitate to reach out on our forums or in our Discord chat.
```
Comment (cannikin) [inline cookbook/Sending_Emails.md:461]: Add links to forums and Discord?

**Review by cannikin — APPROVED** [review]

**Comment by Tobbe** [comment]
I wrote an alternative recipe over here https://github.com/redwoodjs/redwoodjs.com/pull/949
Which one is best? Which one should we keep and publish on rwjs.com?

**Comment by Tobbe** [comment]
Closing this in favor of #949

**Pull request closed without merging**